browser/src/control/Control.JSDialog.js

```diff
@@ -383,15 +383,22 @@ L.Control.JSDialog = L.Control.extend({
 		if (instance.canHaveFocus && initialFocusElement && initialFocusElement.length)
 			initialFocusElement[0].focus();
 
-		var focusWidget = instance.init_focus_id ? instance.container.querySelector('[id=\'' + instance.init_focus_id + '\']') : null;
-		if (focusWidget)
-			focusWidget.focus();
-		if (focusWidget && document.activeElement !== focusWidget) {
-			var firstFocusable = JSDialog.GetFocusableElements(focusWidget);
-			if (firstFocusable && firstFocusable.length)
-				firstFocusable[0].focus();
-			else
-				console.error('cannot get focus for widget: "' + instance.init_focus_id + '"');
+		// Check if the element with id 'tabcontrol' exists in the DOM
+		const isTabControlWidget = document.getElementById('tabcontrol') ? true : false;
```

Comment:
Please do not use hardcoded ids of elements.
This can find any tabcontrol, not in the current dialog too.
If we want, we can scan the tree of widgets (JSON) for widget with `type: tabcontrol`